### DISCUSSION TITLE
Can I work on the same project on different PCs/computers?

### DISCUSSION THREAD

**Discussion by LiseLVan** [asreview/asreview#1633] [Q&A]
I`m using ASReview for the first time.
I downloaded it on my PC at work and on my home laptop.

I started a project on the PC, but I can`t work on the same project on ASreview on my laptop.
Can those two systems connect with each other so I can work on the same project on different computers?

Someone who can help me?

**Comment by Rensvandeschoot** [comment]
The easiest way (if you don't want to use the [server version](https://asreview.readthedocs.io/en/latest/server_overview.html)) is to [export](https://asreview.readthedocs.io/en/latest/manage.html#export-project) a project if you are done at work, and [import](https://asreview.readthedocs.io/en/latest/manage.html#import-project) this project if you want to continue screening on a different device. 

But let's see if others have different suggestions...

**Reply by LiseLVan** [reply]
Thank you for your help!
